tdms/src/tdms_iterator.cpp

```diff
@@ -6087,28 +6090,31 @@ if(runmode == rm_complete && (nvertices>0) )
   }
   
   //now find the maximum absolute value of residual field in the grid
-  for(k=0;k<(K_tot+1);k++)

```

Comment:
Reminder that this includes changes from #32, so we should probably wait for that PR to be merged before going on with this one.

Reply:
💯